test/gtest/p4runtime.cpp

```diff
@@ -1660,6 +1660,36 @@ TEST_F(P4RuntimeDataTypeSpec, Enum) {
     EXPECT_EQ("MBR_2", it->second.members(1).name());
 }
 
+// Currently throws an exception:
```

Comment:
@mbudiu-vmw I could use your help with this test.

I thought I would add a test for serializable enums, just like I wrote a while ago for enums, but the test throws a Compiler Bug in the type checker for some reason. If I write a similar program and run it through `p4test`, there is no issue. The different is that these tests only run the `TypeChecking` pass (see line 1409) and not the whole frontend. Is there another pass we should run? What's so special about serializable enums here?